jspsych-uil-session.js

```diff
@@ -143,5 +146,18 @@ else
 
             api.sessionUpload(access_key, session_id, data);
         };
+
+        /**
+         * Obtain an subject_id from the the dataserver
+         *
+         * @returns the subject_id from the data store.
+         */
+        context.subject_id = function() {
```

Comment:
This function name isn't following the codingConventions.

Reply:
fixed in https://github.com/UiL-OTS-labs/jspsych-uil-utils/pull/17/commits/db45cd64d43aba64a3bac618cf6893fb71f9efe7